tests/playwright-test/types.spec.ts

```diff
@@ -182,3 +182,29 @@ test('config should allow void/empty options', async ({ runTSC }) => {
   });
   expect(result.exitCode).toBe(0);
 });
+
+test('config should allow ExtraUseOptions', async ({ runTSC }) => {
+  const result = await runTSC({
+    'playwright.config.ts': `
+      const configs: pwt.Config[] = [];
+      configs.push({ use: { myParam: 'text' }});
+    `,
+    'extraUseOptions.d.ts': `
+      export {};
+      declare global {
+        namespace PlaywrightTest {
+          export interface ExtraUseOptions {
+            myParam: string
```

Comment:
Does this cause myParam to be mandatory inside use? I think it does. We can convert ExtraUseOptions to an optional type like so:

```ts
type OptionalType = {[x in keyof PlaywrightTest.ExtraUseOptions]?: PlaywrightTest.ExtraUseOptions[x]};
```

Reply:
Probably leave it up to the user? (What if all their tests have this param...)